rpmio/rpmpgp_sequoia.c

```diff
@@ -39,6 +39,9 @@ W(rpmRC, pgpVerifySignature,
 W(int, pgpPubkeyKeyID,
   (const uint8_t * pkt, size_t pktlen, pgpKeyID_t keyid),
   (pkt, pktlen, keyid))
+W(int, pgpPubkeyFingerprint,
+  (const uint8_t * pkt, size_t pktlen, uint8_t **fp, size_t *fplen),
+  (pkt, pktlen, fp, fplen))
```

Comment:
Doh, I would've added this already but totally forgot that we need such a change on rpm side too. And that was even by my request :facepalm::laughing:

Reply:
No worries!